sky/clouds/service_catalog/data_fetchers/fetch_cudo.py

```diff
@@ -9,98 +9,9 @@
 
 import cudo_compute
 
-VMS_CSV = 'cudo/vms.csv'
+import sky.clouds.service_catalog.cudo_catalog_helper as helper
```

Comment:
Need to be changed to the following?
```suggestion
from sky.provision.cudo import cudo_utils
```

Reply:
Also, need to change the usage of this module in this file as well?

Reply:
ok fixed